tailwind.config.cjs

```diff
@@ -2,7 +2,11 @@
 module.exports = {
   content: ['./src/**/*.@(css|tsx)'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        'intavia-green': '#00B050',
```

Comment:
let's call it `brand` or `intavia-brand` - maybe it ends up not being olive green